### PR TITLE
New Section for Paid / Licensed code

### LINKED ISSUE #2946: WordPress Licenses for multi-environment use
Re: <New Doc>

## Issue Description
Users ask recommendation about what to do for plugins that have licenses.

## Suggested Resolution
Personal suggestion is to link in DEV to update, and then link the license in LIVE manually
Any other suggestion is welcome.

### PR DESCRIPTION
**Pull request by eabquina**
New Section about Licenses

Closes #2946

## Effect
PR includes the following changes:
- Paid Plugin and Licenses
-
-

## Remaining Work
- [ ] List any outstanding todos
- [ ] If needed

### CODE REVIEW

File: source/_docs/unsupported-modules-plugins.md [hunk 1]
```diff
@@ -445,6 +445,19 @@ The list of [WordPress roles and capabilities](https://codex.wordpress.org/Roles
 * `edit_plugins`
 * `edit_themes`
 
+## Paid Plugins or Licenses
+
+In general, licensed or paid plugins, modules or themes are allowed for use in Pantheon. For their use, consider platform constrants or conflicts that will arise due to the nature of the platform. Licensing issues are for Customers to Resolve, as per the terms of their plugin or themes.
```
Comment (alexfornuto) [inline source/_docs/unsupported-modules-plugins.md]: "consider platform constrants or conflicts that will arise due to the nature of the platform."

This is ambiguous to the point of not being helpfuil. What constraints specifically affect licensed plugins?

File: source/_docs/unsupported-modules-plugins.md [hunk 2]
```diff
@@ -445,6 +445,19 @@ The list of [WordPress roles and capabilities](https://codex.wordpress.org/Roles
...
+
+Paid tools or licenses usually get direct support from developers. It is also used to get updates directly online. Most of the time, you update your code in DEV, and just commit the changes to TEST and LIVE.
```
Comment (alexfornuto) [inline source/_docs/unsupported-modules-plugins.md]: You want to update code to DEV *all of the time*. This isn't specific to plugins with licenses.

**Comment by rachelwhitton** [comment]
Feels like it might be a better fit in [Platform Considerations](https://pantheon.io/docs/platform-considerations/)?

**Comment by rachelwhitton** [comment]
Also curious if there are Drupal instances of this issue or if it's a WP specific scenario

**Comment by eabquina** [comment]
@alexfornuto @rachelwhitton I agree that this should be in Platform Considerations.  
I created another PR there so to have its own section there. After the intro, I made a link to the unsupported plugins or modules docs (which some of them are paid/licensed) to see a longer list.

**Comment by alexfornuto** [comment]
@eabquina as I asked [there]((https://github.com/pantheon-systems/documentation/pull/2983#issuecomment-335957882) Does that PR replace this one? Please close/consolidate/merge your PRs on this issue so we know what you're ultimately trying to propose.

**Comment by eabquina** [comment]
@alexfornuto thanks for the help on my mistake, I have merged the proposed content and will ask to review instead of having separate PRs

**Comment by rachelwhitton** [comment]
After additional consideration, we're going to close this PR in favor of a new doc to be written as part of work done in #2946 by @alexfornuto

**Pull request closed without merging**